mne/bem.py

```diff
@@ -1108,12 +1107,112 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
 
     # Show computed BEM surfaces
     if show:
-        plot_bem(subject=subject, subjects_dir=subjects_dir,
+        plot_bem(subject=subject, sub1jects_dir=subjects_dir,
                  orientation='coronal', slices=None, show=True)
 
     logger.info('Created %s\n\nComplete.' % (fname_head,))
 
 
+def convert_surface(fname_in, fname_out, T1_mgz, useLAS=False):
+    """Convert surface.
+
+    Parameters
+    ----------
+    fname_in : str
+        Path to surface in FreeSurfer surf format.
+    fname_out : str
+        File to write.
+    T1_mgz : str
+        Full path to T1.mgz file.
+    useLAS :
+        Flip x axis.
+
+    Returns
+    -------
+    surf : dict
+        Converted surface.
+    """
+    from .source_space import _get_mgz_header
+    from .surface import _read_surface_geom, write_surface
+    header = _get_mgz_header(T1_mgz)
+    surf_in = _read_surface_geom(fname_in)
+    surf_in['coord_frame'] = FIFF.FIFFV_COORD_MRI
```

Comment:
And this is ``FIFFV_COORD_UNKNOWN`` in the c-code, but it throws an error in python.